deploy/deployment/openshift/objects/chemanagers.che.eclipse.org.CustomResourceDefinition.yaml

```diff
@@ -71,6 +71,42 @@ spec:
                 of the che operator deployment/pod. If not defined there, it defaults
                 to a hardcoded value.
               type: string
+            k8s:
```

Comment:
you may want to collapse generated stuff by default, as DWO does https://github.com/devfile/devworkspace-operator/blob/main/.gitattributes